src/twisted/names/authority.py

```diff
@@ -34,28 +44,14 @@ def getSerial(filename = '/tmp/twisted-names.serial'):
         os.umask(o)
 
     with open(filename, 'r') as serialFile:
-        lastSerial, ID = serialFile.readline().split()
+        lastSerial, id = serialFile.readline().split()
```

Comment:
Technically (naming-standard-wise) it is correct to complain; it should just also be complaining about `id`.  `zoneID` ought to be OK with it though.